ynr/apps/uk_results/views/base_views.py

```diff
@@ -80,6 +82,29 @@ def get_context_data(self, **kwargs):
 
         return context
 
+    def get_ballot_paper_ids_from_csv(self):
```

Comment:
It might be better to store the modgov URL on the ballot rather than just the CSV, as that mean all we need to check is a field on the ballot?

We could do this by adding a management command that reads from `election_id_to_url.csv` and stores the URL's on the objects.